### DISCUSSION TITLE
[Thesis] Dungeon-Hamster

### DISCUSSION THREAD

**Discussion by cagix** [Programmiermethoden/Dungeon#1371] [Ideas]
Portieren des [Java-Hamster-Framesworks](https://www.java-hamster-modell.de/simulator.html) in unseren Dungeon

**Comment by ghost** [comment]
Meilensteine für dieses Issue:
- [x] https://github.com/Programmiermethoden/PM-Dungeon/pull/417
In diesem Meilenstein wurde die Basis für Charaktere eingeführt, welche jeweils einen eigenen Thread darstellen.
- [x] https://github.com/Programmiermethoden/PM-Dungeon/pull/419
Hier wird der Level-Editor umgesetzt, mit welchem Hamster-Belohnungen, Hindernisse und später auch Hamster selbst im Level platziert werden können
- [ ] https://github.com/Programmiermethoden/PM-Dungeon/pull/422
Hier wird die Game-Klasse für den Hamster-Simulator umgesetzt. Mit allen nötigen Details um eine Demo lauffähig zu machen (Kollisionserkennung, Verwaltung der Objekte etc.). Zudem erfolgt hier die konkrete Umsetzung eines Threaded Characters als Hamster.
